fineract-core/src/main/java/org/apache/fineract/infrastructure/jobs/service/JobName.java

```diff
@@ -58,7 +58,7 @@ public enum JobName {
     PURGE_EXTERNAL_EVENTS("Purge External Events"), //
     PURGE_PROCESSED_COMMANDS("Purge Processed Commands"), //
     ACCRUAL_ACTIVITY_POSTING("Accrual Activity Posting"), //
-    ;
+    ADD_PERIODIC_ACCRUAL_ENTRIES_FOR_SAVINGS_WITH_INCOME_POSTED_AS_TRANSACTIONS("Add Accrual Transactions For Savings"); //
```

Comment:
Is this Job for posting the Income as standalone "Income Posting" transaction type or to create the accrual transactions for the savings account?